CHANGELOG.md

```diff
@@ -2,6 +2,76 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.64.0](https://github.com/aws/aws-cdk/compare/v1.63.0...v1.64.0) (2020-09-22)
+
+
+### ⚠ BREAKING CHANGES TO EXPERIMENTAL FEATURES
+
+* **codedeploy:** the default policy for `LambdaDeploymentGroup` no longer contains `sns:Publish` on `*` permissions
+* **cfn-include:** the construction property 'nestedStacks' of class 'CfnInclude' has been renamed to 'loadNestedStacks'
+* **rds:** removed protected member `subnetGroup` from DatabaseCluster classes
+* **rds:** Cluster now has deletionProtection enabled if its removal policy is `RETAIN`
+* **rds**: Instance now has deletionProtection enabled by default only if its removal policy is `RETAIN`
+* **cfnspec:** Fixed ECS task definition within the L1 layer. Fixed the casing of the `efsVolumeConfiguration` property to match the spec published by cloudformation. Fixed the type of the `DockerVolumeConfiguration.labels` property to allow users to properly apply labels.
```

Comment:
It's not a breaking change if it was broken before

Reply:
So the `labels` thing was indeed broken and agree it isn't really a breaking change. 

The casing problem though wasn't strictly broken, i.e templates that used the "wrong" casing still worked, even though it doesn't match the CloudFormation spec casing. We decided to still change it to be aligned with the spec (the wrong casing was introduced manually by mistake).

See https://github.com/aws/aws-cdk/pull/10201#discussion_r489034275

We decided it's ok to "break" this and add a notice, but now that i think of it, our notice explicitly states: "BREAKING CHANGES TO EXPERIMENTAL FEATURES" - which is not the case. 

@eladb - Any suggestions where to put the notice? We do want users to be aware they are going to see a resource replacement.

Reply:
What is the impact from a user's perspective? Is it a resource replacement? If there is no real impact from a user point of view, I'd just omit from this section. It's scary as it is.

Reply:
Reverted the change.